### ISSUE TITLE
Can Gloo support ECC certs

### ISSUE THREAD

**Issue by cmwylie19** [solo-io/gloo#4536]
# Can Gloo Edge support ECC certs?
_ECC certs are small and efficient, are we able to leverage ECC certs in Gloo?_

**Comment by mlholland** [comment]
Yes, Gloo surfaces Envoy's TLS APIs, which support ECC, or ECDH-curves as Envoy refers to them. Search for ECDH here to read more about it: https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/transport_sockets/tls/v3/common.proto

There's also some documentation on Envoy's multi-cert selection with respect to ECDH here, which is important since they're a little strict about which ECC certs they'll accept: https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/security/ssl
